Tests/ConduitTests/Networking/Serialization/XML/XMLRequestSerializerTests.swift

```diff
@@ -47,6 +58,25 @@ class XMLRequestSerializerTests: XCTestCase {
         XCTAssert(xml != nil)
     }
 
+    func testMalformedXMLSerializesXML() throws {
+        let request = try makeRequest()
+        let serializer = XMLRequestSerializer()
+
+        guard let modifiedRequest = try? serializer.serialize(request: request, bodyParameters: XML(root: textMalformedXML)) else {
+            XCTFail("Serialization failed")
+            return
+        }
+
+        guard let httpBody = modifiedRequest.httpBody,
+            let xmlString = String(data: httpBody, encoding: .utf8) else {
+                XCTFail("Expected body")
+                return
+        }
+
+        let xml = XML(xmlString)
+        XCTAssert(xml != nil)
+    }
```

Comment:
In addition to checking the XML can be serialized and read, please validate that the text value of `<malformed>` has been properly escaped.